src/v0/destinations/google_adwords_offline_conversions/utils.js

```diff
@@ -359,9 +370,21 @@ const getClickConversionPayloadAndEndpoint = (message, Config, filteredCustomerI
   if (!properties.conversionEnvironment && conversionEnvironment !== 'none') {
     set(payload, 'conversions[0].conversionEnvironment', conversionEnvironment);
   }
+
+  // add consent support for click conversions
+  const consentObject = finaliseConsent(eventLevelConsent, Config, consentFields);
+  set(payload, 'conversions[0].consent', consentObject);
   return { payload, endpoint };
 };
 
+const getConsentsDataFromIntegrationObj = (message, conversionType) => {
```

Comment:
why to bring `conversionType` here ? Instead, just pass {} eventLevelConsent object in store conversions code path flow when using same utility functions
That way, code is more readable - otherwise, unless one pays attention to this check here, it give impression that store conversions also consider eventLevelConsents.

Reply:
edited